v3/src/models/data/data-set.ts

```diff
@@ -209,12 +209,19 @@ export const DataSet = types.model("DataSet", {
   let _childCases: IGroupedCase[] = []
   let isValidCollectionGroups = false
 
-  function getCollection(collectionId: string): ICollectionModel | undefined {
+  function getRealCollection(collectionId: string): ICollectionModel | undefined {
```

Comment:
How about `getGroupedCollection()`?

Reply:
Sounds good. I didn't like the "real" naming, I just couldn't come up with anything better. I just pushed a commit that changes this name.